javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,46 @@
+
+//Flight booking fullname function
+
+function getFullName(firstName, surname, useFormalName = true, gender = "") {
+    if (useFormalName) {
+        if (gender === "male") {
+            return "lord " + firstName + surname;
+        } else if (gender === "female") {
+            return "lady " + firstName + surname;
+        }
+    }
+            return firstName + surname;
+        }
+    
+
+    console.log(getFullName("Benjamin ", "Hughes", true,"male"))
+    console.log(getFullName("Benjamin ", "Hughes", false))
+    console.log(getFullName("Stefany"," Romero",true,"female"))
+
+
+//Event Application
+
+function getEventWeekday(daysFromToday){
+
+const weekDays= [,"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+const today = new Date().getDate();
+const eventDay = (today + daysFromToday) % 7;
+return weekDays[eventDay];
+
+}
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(2)); 
+
+//Weather wear
+
+function clothesTowear(temperature){
+    
+    if( temperature >= 15){
+        return  "temperature is " + temperature + " you sholud wear" + "Jacket and pants" 
+    }else{
+        return  "temperature is " + temperature + " you sholud wear"+ "shorts and t shirt"
```

Comment:
The outcome of the function will be `temperature is 18 you sholud wearJacket and pants` and there is no space between wear and Jacket.
Also it can works exactly the same if you just combine them `return  "temperature is " + temperature + " you should wear shorts and t shirt"`
Otherwise you can use template literals like this: `temperature is ${temperature} you should wear T-shirt and shorts`;